src/main/scala/com/vngrs/etl/transformers/RowToJsonTransform.scala

```diff
@@ -0,0 +1,28 @@
+package com.vngrs.etl.transformers
+
+import com.vngrs.etl.Transform
+import org.apache.spark.rdd.RDD
+import org.apache.spark.sql.{Row, SQLContext}
+
+/**
+  * Transform [[org.apache.spark.sql.Row]]s to `JSON` formatted [[String]]s.
+  */
+final case class RowToJsonTransform() extends Transform[Row, String] {
+
+  /**
+    * Transform [[org.apache.spark.sql.Row]]s to `JSON` formatted [[String]]s.
+    *
+    * @param input [[org.apache.spark.rdd.RDD]] of [[org.apache.spark.sql.Row]]s
+    * @return Transformed [[org.apache.spark.rdd.RDD]] of `JSON` [[String]]s
+    */
+  override def apply(input: RDD[Row]): RDD[String] = {
+    val sqlCtx = new SQLContext(input.sparkContext)
+
+    val df = input.take(1).headOption.map(_.schema) match {
+      case Some(schema) => sqlCtx.createDataFrame(input, schema)
+      case None         => sqlCtx.emptyDataFrame
```

Comment:
You are right about logs. However, I wish to talk about logs before adding log mechanism to the project.

Aside from logging mechanism, do you think a warning log is enough? Or should we throw an exception?